docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -419,14 +419,109 @@ As discussed in treatment guideline reviews, in general triplet regimens are pre
      - High
      - VRd: bortezomib + lenalidomide + dexamethasone
 
-First-line treatment
-^^^^^^^^^^^^^^^^^^^^
+Treatment Modeling Strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Second-line treatment
-^^^^^^^^^^^^^^^^^^^^
+**Scope**:
+
+This modeling strategy allows us to track the proportion of MM/RRMM simulants that receive an isatuxamib-containing treatment. It also allows for coherent assignment of isatuxamib-containing treatments at the simulant level so that any given simulant is not assigned an isatuxamib-containing treatment an illogical amount of times, which will reduce noise in our estimations.
+
+This model strategy does NOT consider:
+
+- Differences between isatuxamib-containing treatment regimens (ex: isatuxamib + pomalidomide + dexamethasone versus isatuxamib + carfilzomib + dexamethasone)
+- Differences between the numerous treatment regimens that do NOT contain isatuxamib (note this has important implications for applying efficacy data to the appropriate reference group(s))
+- Specific eligibility requirements for isatuxamib-contianing treatments other than previous treatment with isatuxamib. Specifically, 
+	- We do not directly consider the requirement of previous treatment with lenalidomide and a PI, although this is likely minimally impactful as these are very common treatment regimens and "almost all" patients with myeloma develop disease that is eventually refracotory to lenalidomide and PIs, [Attal-et-al-2019]_.
+	- We do not consider previous treatment with other monoclonal antibody drugs such as daratumumab, which may impact probability of receiving an isa-containing treatment
```

Comment:
No previous treatment with Dara doesn't mean you couldn't get Dara at same line as Isa, right?

Reply:
I meant if you had dara in a previous line, you're less likely to be prescribed isa in a subsequent line. I don't think dara and isa would ever be given at the same time.